src/pages/createHost.js

```diff
@@ -26,8 +25,7 @@ class CreateHostPage extends React.Component {
   }
 
   async componentDidMount() {
-    const { getLoggedInUser } = this.props;
-    const LoggedInUser = getLoggedInUser && (await getLoggedInUser());
```

Comment:
If component gets updated later (which will happen because `LoggedInUser` is fetched after the mount) then we won't get the updated `LoggedInUser`. We don't need to store the user in the state anymore, we can rely on `this.props` directly.